### PR TITLE
Bind_event_listener_when_registering_display_object

### PR DESCRIPTION
**Pull request by sudhir-jedhe**

### CODE REVIEW

File: src/widgets/CheckBox.js [hunk 1]
```diff
@@ -11,14 +11,32 @@ export default class CheckBox extends createjs.Container {
...
+      ]
     });
     this._createAsset();
   }
 
   _createAsset() {
     this._addBoxArea();
     this._addCheckMark();
-    this._addInteraction();
+    // this._addInteraction();
```
Comment (mnewcomb314) [inline src/widgets/CheckBox.js]: Nit: please delete stale code

File: src/widgets/CheckBox.js [hunk 2]
```diff
@@ -59,12 +77,12 @@ export default class CheckBox extends createjs.Container {
...
-    this.addEventListener('keyboardClick', this.onChange.bind(this));
-    this.addEventListener('focus', this.onFocus.bind(this));
-    this.addEventListener('blur', this.onBlur.bind(this));
-  }
+  // _addInteraction() {
+  //   this.addEventListener('mousedown', this.onChange.bind(this));
+  //   this.addEventListener('keyboardClick', this.onChange.bind(this));
+  //   this.addEventListener('focus', this.onFocus.bind(this));
+  //   this.addEventListener('blur', this.onBlur.bind(this));
+  // }
```
Comment (mnewcomb314) [inline src/widgets/CheckBox.js]: Nit: please delete stale code

File: src/widgets/Button.js [hunk 3]
```diff
@@ -17,6 +17,28 @@ export default class Button extends createjs.Container {
       displayObject: this,
       accessibleOptions: options,
       role: AccessibilityModule.ROLES.BUTTON,
+      events: [
```
Comment (mnewcomb314) [inline src/widgets/Button.js:20]: I like this better than what I was thinking of for event registration!  And it has the added flexibility of allowing for binding multiple listeners for the same event.

**Comment by mnewcomb314** [comment]
Looks good!  Just some minor nits, and I'm also holding this PR until the CAM `1.0.0-prerelease.3` is done which will be after your corresponding CAM PR merges (and once that build exists, I will ask you to please update package*.json to point to that version of CAM).

File: src/widgets/CheckBox.js (hunk 2 above)
Comment (sudhir-jedhe) [reply src/widgets/CheckBox.js]: removed

File: src/widgets/CheckBox.js (hunk 1 above)
Comment (sudhir-jedhe) [reply src/widgets/CheckBox.js]: removed

**Comment by sudhir-jedhe** [comment]
@mnewcomb314  updated CAM version

**Review by mnewcomb314 — APPROVED** [review]

**Pull request merged**